### PR TITLE
[IMP] l10n_{ar,cl,ec,pe}: improve tax names, descriptions, and translations

### PR DESCRIPTION
**Pull request by aliyatastemirova**
Before the change
- Tax Names were not intuitive or easily understandable for the users, it was hard to relate which tax was referenced with each name.
- The Name didn't include the Spanish translation, regardless the name is encoded now, it should be encoded based on the Spanish description.

After the change:
- Tax names(codes) both in English and Spanish are more intuitive and easier to understand for the users.
- Tax names have better matching with the description.

task-3551393

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by vbe-odoo — COMMENTED** [review]
@aliyatastemirova 
I have only comments on 1 file for AR.
EC looks good, I've checked each one of them and they look OK.
Thanks!
cc' @ren-odoo @fvz-odoo

File: addons/l10n_cl/data/template/account.tax-cl.csv [hunk 1]
```diff
@@ -1,101 +1,101 @@
...
+"","","","","","","","","","","","tax","invoice","+IVA Pagado No Recuperable","account_420220","","","",""
+"","","","","","","","","","","","base","refund","-Compras IVA No Recuperable","","","","",""
+"","","","","","","","","","","","tax","refund","-IVA Pagado No Recuperable","account_420220","","","",""
+"iva_compra_uso_comun","19% C","VAT Purchase 19% Common Use","VAT 19% purchase for public use","19.0","percent","purchase","14","tax_group_iva_19","","6","base","invoice","+Compra Netas Gravadas Con IVA Uso Comun","","","IVA Compra 19% Uso Común","19% IVA UC","IVA 19% CUC"
+"","","","","","","","","","","","tax","invoice","+IVA Pagado Compras Uso Común","account_110730","","","",""
+"","","","","","","","","","","","base","refund","-Compra Netas Gravadas Con IVA Uso Comun","","","","",""
+"","","","","","","","","","","","tax","refund","-IVA Pagado Compras Uso Común","account_110730","","","",""
+"iva_supermercado_recup","19% SM","VAT Purchase 19% Supermarket","VAT 19% supermarket recup","19.0","percent","purchase","14","tax_group_iva_19","","6","base","invoice","+Compras De Supermercado","","","IVA Compra 19% Supermarket","19% IVA SupMRec","IVA 19% SupMRec"
```
Comment (fvz-odoo) [inline addons/l10n_cl/data/template/account.tax-cl.csv:98]: Hey @aliyatastemirova 

Thanks for the PR, on my side (and this was my mistake, sorry!) we should only translate the part that says "IVA Compra 19% Supermarket" to "IVA Compra 19% Supermercado Recup".

Thanks a lot!

Fernando

File: addons/l10n_pe/data/template/account.tax-pe.csv [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_ina","0% Ina","0% Inaffected Purchase","INA","9998","Z","0.0","purchase","1","","tax_group_ina","","base","invoice","","","0% Ina","0% Inafectado"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_gra","0% Gra","0% Free Purchase","GRA","9996","E","0.0","purchase","1","","tax_group_gra","","base","invoice","","","","0% Gratis"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"sale_tax_exp","0% Exp","0% Exportation Sale","EXP","9995","S","0.0","sale","1","1","tax_group_exp","","base","invoice","","0","","0% EXP"
```
Comment (fvz-odoo) [inline addons/l10n_pe/data/template/account.tax-pe.csv:58]: And just a small one here @aliyatastemirova the part "0% EXP" at the end of the line should be "0% Exp". 

Thanks!

Fernando

**Comment by fvz-odoo** [comment]
Hello @aliyatastemirova 

I tested l10n_cl and l10n_pe and both LGTM. 

Thanks

Fernando

@ren-odoo @vbe-odoo

**Comment by vbe-odoo** [comment]
Hey @aliyatastemirova thanks for the changes.
I've checked l10n_ar and l10n_ec and everything LGTM now!
Thanks!
cc' @fvz-odoo @ren-odoo

File: addons/l10n_pe/data/template/account.tax-pe.csv [hunk 3]
```diff
@@ -1,61 +1,61 @@
...
+"","","","","","","","","","","","","tax","refund","chart40116","","",""
+"purchase_tax_exp_0","0% Exp","0% Exportation Purchase","EXP","9995","S","0.0","purchase","1","1","tax_group_exp","True","base","invoice","","","0% Exp","0% Exp"
+"","","","","","","","","","","","","tax","invoice","chart40115","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40115","","",""
+"purchase_tax_exo","0% Exo","0% Exonerated Purchase","EXO","9997","E","0.0","purchase","1","","tax_group_exo","","base","invoice","","","0% Exo","0% Exonerado"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_ina","0% Ina","0% Inaffected Purchase","INA","9998","Z","0.0","purchase","1","","tax_group_ina","","base","invoice","","","0% Ina","0% Inafectado"
```
Comment (jco-odoo) [inline addons/l10n_pe/data/template/account.tax-pe.csv:50]: is not it Unaffected?

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**